src/main/java/uk/sky/cqlmigrate/ExecutionInfo.java

```diff
@@ -0,0 +1,29 @@
+package uk.sky.cqlmigrate;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.Session;
+
+class ExecutionInfo {
```

Comment:
A better name for this would be something like `SessionContext` I think, to indicate it's about the cassandra session. I think execution info is too generic.